ezcontent.libraries.yml

```diff
@@ -0,0 +1,10 @@
+ezcontentjs:
```

Comment:
We can move these file on block, level and change the library and js file name specific to that block and feature.